pwsh-extension-dev.code-workspace

```diff
@@ -0,0 +1,493 @@
+        ],
+        "group": {
+          "kind": "test",
+          "isDefault": true
+        }
+      },
+      {
+        "label": "Test Server",
+        "type": "shell",
+        "options": {
+          "cwd": "${workspaceFolder:Server}"
+        },
+        "problemMatcher": [
+          "$msCompile"
+        ],
+        "command": "Invoke-Build TestPS74",
+        "group": {
+          "kind": "test",
+          "isDefault": true
+        }
+      },
+      {
+        "label": "Invoke-Build Client",
+        "type": "shell",
+        "options": {
+          "cwd": "${workspaceFolder:Client}"
+        },
+        "command": "Invoke-Build ${input:clientBuildCommand}",
+        "group": "build"
+      },
+      {
+        "label": "Invoke-Build Server",
+        "type": "shell",
+        "options": {
+          "cwd": "${workspaceFolder:Server}"
+        },
+        "command": "Invoke-Build ${input:serverBuildCommand}",
+        "group": "build"
+      }
+    ],
+    "inputs": [
+      {
+        "type": "pickString",
+        "id": "clientBuildCommand",
+        "description": "Which Invoke-Build Client Task?",
+        "options": [
+          "Restore",
+          "Clean",
+          "Build",
+          "Test",
+          "Package"
+        ],
+        "default": "Clean"
+      },
+      {
+        "type": "pickString",
+        "id": "serverBuildCommand",
+        "description": "Which Invoke-Build Server Task?",
+        "options": [
+          "SetupDotNet",
+          "BinClean",
+          "Clean",
+          "Build",
+          "Test",
+          "TestPS74",
+          "TestE2EPwsh",
+          "TestPS51",
+          "TestE2EPowerShell",
+        ],
+        "default": "Clean"
+      }
+    ],
+
+  },
+  "launch": {
+    "version": "0.2.0",
+    "compounds": [
+      {
+        "name": "Launch Extension",
+        "configurations": [
+          "Launch",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 1
+        }
+      },
+      {
+        "name": "Launch Extension - Temp Profile",
+        "configurations": [
+          "Launch-Temp",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "PreLaunch",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 2
+        }
+      },
+      {
+        "name": "Launch Extension - Isolated Profile",
+        "configurations": [
+          "Launch-Isolated",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 3
+        }
+      },
+      {
+        "name": "Interactively Test Rename",
+        "configurations": [
+          "Launch Extension - Rename Test Cases",
```

Comment:
Huh, what's this one do?

Reply:
Sorry, it's a relic from the skipped-worktree code-workspace I was using to work on the rename PR. It will land as part of that one. Good Catch!